esprit-woo/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+node_modules/
```

Comment:
minor: duplicate

Reply:
probably you could ignore the `build` folder as well? don't really bother changing it tho.

Reply:
Yeah so the plugin (in php) will read the javascript from the single build file, which was built using the @wordpress/scripts npm package. That's how the php plugin can use npm modules. Just sharing since I thought it was interesting.